index.html

```diff
@@ -4,9 +4,53 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
-    <link rel="stylesheet" href="index.css" />
+    <link rel="stylesheet" href="/css/index.css" />
+    <link href="https://unpkg.com/boxicons@2.0.9/css/boxicons.min.css" rel="stylesheet" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@400;700&family=Roboto:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
+
   <body>
-    
+    <header>
+      <div class="mobile-header">
+        <div class="logo">
+          <ul>
+            <li>
+              <a class="button" href="/index.html"><img src="./img/mrtaaj.png" alt="mrtaaj"/></a>
+            </li>
+          </ul>
```

Comment:
This is just a heads up. Lists are used when we have multiple items to display. This is just one item so I'll recommend we just have the `<a>` tag there and remove the `<ul><li>`.